### PR TITLE
upstream CI: Enable CentOS 8 Stream for PR and nightly tests.

### PR DESCRIPTION
**Pull request by rjeffman**
Add configuration to execute build a testing CentOS 8 stream image and
to execute upstream tests using that image in pull requests (Ansible
2.9) and on the nightly tests (all supported Ansible versions).

### CODE REVIEW

File: tests/azure/build-containers.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 schedules:
-- cron: "0 0 * * 0"
+- cron: "0 0 * * *"
```
Comment (t-woerner) [inline tests/azure/build-containers.yml:4]: Why have you added this change?

**Comment by t-woerner** [comment]
I would like to have an extra PR for the c9s changes. With c8s there is different naming schema. What do you think about renaming centos-9* to c9s* in an extra PR?

**Comment by rjeffman** [comment]
I agree, and every time I revisit this PR I find it annoying that versions 9 and 8 have different names, and the name for version 9 don't show what it really is.

**Comment by rjeffman** [comment]
@t-woerner I opened PR #770 to rename 'centos-9' to 'c9s'.

File: molecule/c8s-build/molecule.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      - 8.8.8.8
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    command: /usr/sbin/init
+    privileged: true
+provisioner:
+  name: ansible
+  playbooks:
+    prepare: ../resources/playbooks/prepare-build.yml
+prerun: false
```
Comment (t-woerner) [inline molecule/c8s-build/molecule.yml:19]: Is is needed to deactivate prerun for build also?

**Review by t-woerner — APPROVED** [review]
LGTM

**Pull request merged**